integrations/tensorflow/compiler/ConvertToMHLO.cpp

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "integrations/tensorflow/compiler/Passes.h"
+#include "compiler/Passes.h"
```

Comment:
This is not great. We probably want to deepen the directory path by one level so we have a unique-ish top-level path segment. ie. move everything in integrations/tensorflow down by one level under a new `iree_tensorflow` directory.

Can do in a follow-up if needed, I guess.

Reply:
Yes, agreed. I was just doing the changes to make it build, but I don't like this at all

Reply:
How about just renaming the `compiler/` directory to `iree_tf_compiler`?